runtime/doc/change.txt

```diff
@@ -386,6 +388,10 @@ octal and hexadecimal numbers and alphabetic characters.  This depends on the
   under or after the cursor.  This is useful to make lists with an alphabetic
   index.
 
+For decimals a leading negative sign is considered for incrementing/
```

Comment:
incrementing or decrementing. Slash could be fine replacement for “or” on one line, but not here.